src/Microsoft.OpenApi/Validations/Rules/OpenApiDocumentRules.cs

```diff
@@ -28,15 +28,6 @@ public static class OpenApiDocumentRules
                             String.Format(SRResource.Validation_FieldIsRequired, "info", "document"));
                     }
                     context.Exit();
-
```

Comment:
can you elaborate a bit on this change please?

Reply:
According to the 3.1 spec, the OpenAPI document MUST at least contain one `paths`, `components` or `webhooks` field hence paths is [no longer a required field](https://spec.openapis.org/oas/v3.1.0#openapi-object). 

However, as I'm typing this I've just realized that removing this would affect version 2.0 and 3.0 as well. Let me add a guard clause that ensures the rule is upheld for 2.0 and 3.0 but ignored for 3.1.